### PR TITLE
Send nations_producer to ATI when given

### LINKED ISSUE #9787: Return nations_producer to ATI on article amp pages
**Is your feature request related to a problem? Please describe.**
Related to https://github.com/bbc/fabl-modules/issues/2413 and https://github.com/bbc/web/issues/6538

- Ares sends `nations_producer` as part of the `analyticsLabels` in the metadata. This is the home nation that produced the content. This needs to be sent on to ATI.

**Describe the solution you'd like**
- Pass the `nations_producer` value from ares (if returned) to ATI

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Checklist**
- [x] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

### PR DESCRIPTION
**Pull request by iainrj**
Resolves #9787 

**Overall change:**
_A very high-level summary of easily-reproducible changes that can be understood by non-devs._
Ares sends a value which describes the home nation editorial team that created the content. This PR sends this value to ATI as part of the article metadata.

**Code changes:**
- Retrieve the value from the article metadata
- Add the correct key for ATI to find and the value from above to the ATI config

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [x] This PR requires manual testing

### Test asset
`c0g992jmmkko`

e.g http://localhost:7080/news/articles/c0g992jmmkko.amp

Navigate to the test asset, and the event should be visible in the ATI tag inspector in the `x10` key.

### CODE REVIEW

File: data/news/articles/c0g992jmmkko.json [hunk 1]
```diff
@@ -672,7 +672,9 @@
     }
   },
   "metadata": {
-    "analyticsLabels": {},
+    "analyticsLabels": {
+      "nations_producer": "scotland"
+    },
```
Comment (HarveyPeachey) [inline data/news/articles/c0g992jmmkko.json:677]: Just to check, this new producer field is only for Optimo articles right?

Comment (iainrj) [reply data/news/articles/c0g992jmmkko.json:677]: Yep only public service Optimo articles (produced by one of the 4 nations teams)

File: src/app/lib/analyticsUtils/article/index.js [hunk 2]
```diff
@@ -24,3 +24,11 @@ export const getLanguage = articleData =>
...
 export const getPromoHeadline = articleData =>
   pathOr(null, ['promo', 'headlines', 'seoHeadline'], articleData);
+
+export const getNationsProducer = articleData => {
+  return pathOr(
+    null,
+    ['metadata', 'analyticsLabels', 'nations_producer'],
+    articleData,
+  );
+};
```
Comment (ryanmccombe) [inline src/app/lib/analyticsUtils/article/index.js:34]: These ramda functions support currying
```suggestion
export const getNationsProducer = pathOr(
  null,
  ['metadata', 'analyticsLabels', 'nations_producer'],
);
```

**Review by ryanmccombe — APPROVED** [review]
lgtm!  👍

**Review by HarveyPeachey — APPROVED** [review]
LGTM

**Review by tochwill — APPROVED** [review]

**Comment by LilyL0u** [comment]
Tested on the suggested asset http://localhost:7080/news/articles/c0g992jmmkko.amp

nations_producer in the metadata is [scotland]
Screenshot 2022-01-19 at 12 26 35

This value can be seen in #10 in tag inspector and also in the network calls

Screenshot 2022-01-19 at 12 26 47
Screenshot 2022-01-19 at 12 29 49


Do you want any other assets tested? I don't know the requirements for which asset types are affected.

**Comment by iainrj** [comment]
Thanks a lot @LilyL0u!

> Do you want any other assets tested? I don't know the requirements for which asset types are affected.

it should just affect public service Optimo articles (produced by one of the 4 nations teams) so I think this should be fine. (@jimjohnsonrollings?)

**Comment by jimjohnsonrollings** [comment]
We should also test an asset that doesn't have a value set, e.g. /news/articles/crq8j409e3qo.amp (on Test), where we'd expect the x10 value to be absent from the ATI call.

**Comment by LilyL0u** [comment]
@jimjohnsonrollings @iainrj 

I checked the asset provided by Jim above that we expect to not have the ATI value. I did it by making fixture data for that asset and looking at it while running this branch.

I do not see any nations_producer value sent to ATI in the Tag Inspector or the network tab:
x10/#10 are not present
Screenshot 2022-01-19 at 16 50 58
Screenshot 2022-01-19 at 16 51 25

**Comment by iainrj** [comment]
I can't see the UI Tests (I get Not Found) could someone who can, have a look and let me know if it's all good?

**Pull request merged**